tempto-ldap/src/main/java/com/teradata/tempto/fulfillment/ldap/OpenLdapObjectDefinitions.java

```diff
@@ -31,31 +32,143 @@
     public static final LdapObjectDefinition TEST_GROUP =
             LdapObjectDefinition.builder("TestGroup")
                     .setDistinguishedName("cn=TestGroup,ou=Test,dc=tempto,dc=com")
-                    .setAttributes(testGroupAttributes())
+                    .setAttributes(getTestGroupAttributes())
                     .addObjectClasses(Arrays.asList("groupOfNames"))
-                            .build();
+                    .build();
+
+    public static final LdapObjectDefinition TEST_USER =
+            LdapObjectDefinition.builder("TestUser")
+                    .setDistinguishedName("uid=testuser,ou=Test,dc=tempto,dc=com")
+                    .setAttributes(getTestUserAttributes())
+                    .addObjectClasses(Arrays.asList("person", "inetOrgPerson"))
+                    .build();
 
-    private static Map<String, String> testGroupAttributes()
+    private static Map<String, String> getTestGroupAttributes()
     {
         Map<String, String> attributes = newHashMap();
         attributes.put("cn", "Test Group");
         attributes.put("member", "uid=testuser,ou=Test,dc=tempto,dc=com");
         return attributes;
     }
 
-    public static final LdapObjectDefinition TEST_USER =
-            LdapObjectDefinition.builder("TestUser")
-                    .setDistinguishedName("uid=testuser,ou=Test,dc=tempto,dc=com")
-                    .setAttributes(testUserAttributes())
-                    .addObjectClasses(Arrays.asList("person", "inetOrgPerson"))
-                            .build();
-
-    private static Map<String, String> testUserAttributes()
+    private static Map<String, String> getTestUserAttributes()
     {
         Map<String, String> attributes = newHashMap();
         attributes.put("cn", "Test User");
         attributes.put("sn", "User");
         attributes.put("password", "testp@ss");
         return attributes;
     }
+
+    public static final LdapObjectDefinition AMERICA_ORG =
+            LdapObjectDefinition.builder("AmericaOrg")
+                    .setDistinguishedName("ou=America,dc=presto,dc=testldap,dc=com")
+                    .setAttributes(newHashMap())
+                    .addObjectClasses(Arrays.asList("organizationalUnit"))
+                    .build();
+
+    public static final LdapObjectDefinition ASIA_ORG =
```

Comment:
These are objects specific to `presto-tests`. Those should not be defined in `tempto` projects but in `presto-product-tests` module of presto.